sre-recipes/sandboxctl

```diff
@@ -179,13 +179,13 @@ def describe():
     gcp_path = "https://console.cloud.google.com"
     project_id, err = utils.get_project_id()
     if err:
-        print("Failed to get project ID:". err)
+        print("Failed to get project ID:", err)
     external_ip, err = utils.get_external_ip()
     if err:
-        print("Failed to get external ID:". err)
+        print("Failed to get external ID:", err)
     loadgen_ip, err = utils.get_loadgen_ip()
     if err:
-        print("Failed to get loadgen ID:". err)
+        print("Failed to get loadgen ID:", err)
```

Comment:
this change should happen in a feature PR, not as part of the release. Otherwise this won't be on the `develop` branch (and may be overwritten on the next release)